### ISSUE TITLE
Show velocity for milestone on burndown chart

### ISSUE THREAD

**Issue by adamreisnz** [ZenHubIO/support#986]
Can we display somewhere the average velocity of the milestone we're viewing on the burndown chart? E.g.:

![image](https://cloud.githubusercontent.com/assets/490562/21258334/54c989f2-c3e1-11e6-9db9-6a73194d57df.png)

We know the number of story points completed = 116
We know the time that has passed = 6
Average velocity = **19.3 points/day**

Can we display this number somewhere, as well with what the "ideal" velocity should be for this milestone? (total points / total days)

**Comment by azenMatt** [comment]
Yes! this is a really popular feature request that we hope to take care of soon - il keep you updated

**Comment by adamreisnz** [comment]
Excellent, thank you
